test_util/aws.py

```diff
@@ -424,6 +424,7 @@ class VpcCfStack(CfStack):
     @classmethod
     def create(cls, stack_name, instance_type, instance_os, instance_count,
                admin_location, key_pair_name, boto_wrapper):
+
```

Comment:
The problem with this is, it is difficult to keep looking for style changes only and that same time wait for future to do only the style changes. I will make this in two PR.  We can find it uncomfortable to review and approve a new line PR. :)